File: src/fractal/world/fractum/_ident.py
"""
Fractum Ident
=============

"""

from typing import Sequence


class FractumIdent(
    tuple[int, ...],
):
    def __init__(
        self,
        ident: Sequence[int],
    ):
        super().__new__(
            tuple,
            ident,
        )

File: pysrc/fractal/world/time/step/_ident.rs
\\\!
Timestep Ident
==============

\\\!

from typing import Sequence


pub struct TimestepIdent(
    tuple<i32, ...>,
):
    fn __init__(
        &self,
        ident: Sequence<i32>,
    ):
        super().__new__(
            tuple,
            ident,
        )